### PR TITLE
chore: re-introduce public.content_preference

### PR DESCRIPTION
**Pull request by vpol**

### CODE REVIEW

File: .infra/clickhouse-sync.yml [hunk 1]
```diff
@@ -71,6 +71,9 @@ max.batch.size: 32000
 metrics.enable: "false"
 cli.port: 7052
 
+# let's try something scary
+single.threaded: "true"
```
Comment (omBratteng) [inline .infra/clickhouse-sync.yml:75]: 😂

**Review by omBratteng — COMMENTED** [review]
Lets give the deployment some more memory, temporarily

**Comment by vpol** [comment]
> Lets give the deployment some more memory, temporarily

I'll update yaml manually if that would be the case.

**Review by omBratteng — APPROVED** [review]

File: .infra/clickhouse-sync.yml (hunk 1 above)
Comment (capJavert) [reply .infra/clickhouse-sync.yml:75]: why single threaded?

Comment (vpol) [reply .infra/clickhouse-sync.yml:75]: 👻 🎃

Comment (vpol) [reply .infra/clickhouse-sync.yml:75]: > why single threaded?

testing performance.

**Pull request merged**